sdk/instrumentation/instrumentation-opentelemetry/review/instrumentation-opentelemetry.api.md

```diff
@@ -0,0 +1,23 @@
+## API Report File for "@azure/instrumentation-opentelemetry"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AzureLogger } from '@azure/logger';
+import { Instrumentation } from '@opentelemetry/instrumentation';
+import { InstrumentationConfig } from '@opentelemetry/instrumentation';
+
+// @public
+export interface AzureSDKInstrumentationOptions extends InstrumentationConfig {
```

Comment:
This is the first time we're seeing AzureSDKInstrumentation with the SDK in naming. We can make it more consistent.